package.json

```diff
@@ -0,0 +1,157 @@
+{
+  "name": "trampss-todomvc",
+  "version": "0.1.0",
+  "description": "TodoMVC with React / Redux / Redux-Saga",
+  "license": "MIT",
+  "scripts": {
+    "start": "cross-env NODE_ENV=development webpack-dev-server --host 0.0.0.0 --inline --hot --port=3000 --history-api-fallback --watch-poll=1000",
+    "build": "cross-env NODE_ENV=production webpack -p",
+    "lint": "eslint --cache --ext js,jsx src",

```

Comment:
Look at this : https://github.com/Zenika/react-workbench/blob/master/package.json#L13

Reply:
👌